diff --git a/node_modules/@wavedrom/doppler/lib/render-memu.js b/node_modules/@wavedrom/doppler/lib/render-memu.js
index 648a2b1..0196281 100644
--- a/node_modules/@wavedrom/doppler/lib/render-memu.js
+++ b/node_modules/@wavedrom/doppler/lib/render-memu.js
@@ -4,7 +4,7 @@ const genSVG = require('onml/gen-svg.js');
 const stringify = require('onml/stringify.js');
 
 
-const renderMenu = () => {
+const renderMenu = (url) => {
 
   const qMark = ['g', {w: 24, h: 24},
     ['circle', {style: 'fill: #facd6a26', r: 12, cx: 12, cy: 12}],
@@ -18,7 +18,7 @@ const renderMenu = () => {
   svg.push(qMark);
 
   const menu = ['a', {
-    href: 'https://github.com/wavedrom/vcdrom/blob/trunk/help.md',
+    href: url,
     target: '_blank'
   }, svg];
 
