diff --git a/node_modules/waveql/lib/mount-codemirror6.js b/node_modules/waveql/lib/mount-codemirror6.js
index 315df78..bfd1e90 100644
--- a/node_modules/waveql/lib/mount-codemirror6.js
+++ b/node_modules/waveql/lib/mount-codemirror6.js
@@ -12,7 +12,7 @@ const extWaveqlLangWith = require('./ext-waveql-lang-with.js');
 const extChangeWith = require('./ext-change-with.js');
 const waveqlCompletionsWith = require('./waveql-completions-with.js');
 
-const mountCodeMirror6 = (sidebar, deso, pstate, render) => {
+const mountCodeMirror6 = (sidebar, deso, pstate, render, plugins = []) => {
   const sidebarState = EditorState.create({
     doc: deso.waveql,
     extensions: [
@@ -31,7 +31,7 @@ const mountCodeMirror6 = (sidebar, deso, pstate, render) => {
       extReadOnlyTogglerWith('Alt-/', true),
       // extEdiatableTogglerWith('Alt-/', true),
       gutter({class: 'cm-wd-gutter'})
-    ]
+    ].concat(plugins)
   });
 
   const view = new EditorView({state: sidebarState, parent: sidebar});
@@ -86,7 +86,8 @@ const mountCodeMirror6 = (sidebar, deso, pstate, render) => {
       //   // }
       // });
     },
-    hasFocus: () => view.hasFocus // cm.hasFocus()
+    hasFocus: () => view.hasFocus, // cm.hasFocus()
+    getView: () => { return view; },
   };
 
 };
